packages/front-end/components/Experiment/SRMWarning.tsx

```diff
@@ -1,124 +1,196 @@
+                  <ul>
+                    <li>Inconsistent ramping of variations</li>
+                    <li>Pausing variations during execution</li>
+                    <li>Injection attacks and hacks</li>
+                  </ul>
+                </li>
+              </ul>
+              <p>
+                <a
+                  target="_blank"
+                  rel="noreferrer"
+                  href="https://exp-platform.com/Documents/2019_KDDFabijanGupchupFuptaOmhoverVermeerDmitriev.pdf"
+                >
+                  Read about SRM issues (PDF)
+                </a>
+              </p>
+            </>
           )}
-        </p>
-        <p>
-          This is a very strong indication that something is fishy with the
-          results. Below are the most common causes:
-        </p>
-        <ul>
-          <li>
-            Bucketing
-            <ul>
-              <li>Bad randomization function</li>
-              <li>Corrupted user IDs</li>
-              <li>Carry over effects from previous tests</li>
-              <li>Interaction effects</li>
-            </ul>
-          </li>
-          <li>
-            Execution
-            <ul>
-              <li>Different start times for variations</li>
-              <li>Variation-specific errors or crashes</li>
-              <li>Variation-specific performance issues</li>
-              <li>Broken event firing</li>
-            </ul>
-          </li>
-          <li>
-            Analysis
-            <ul>
-              <li>Broken filtering (e.g. bot removal)</li>
-              <li>Missing data</li>
-              <li>Wrong start date</li>
-              <li>Wrong triggering condition</li>
-            </ul>
-          </li>
-          <li>
-            Interference
-            <ul>
-              <li>Inconsistent ramping of variations</li>
-              <li>Pausing variations during execution</li>
-              <li>Injection attacks and hacks</li>
-            </ul>
-          </li>
-        </ul>
-        <p>
-          If you are interested, there is a detailed write-up of common SRM
-          issues at{" "}
-          <a
-            target="_blank"
-            rel="noreferrer"
-            href="https://exp-platform.com/Documents/2019_KDDFabijanGupchupFuptaOmhoverVermeerDmitriev.pdf"
-          >
-            https://exp-platform.com/Documents/2019_KDDFabijanGupchupFuptaOmhoverVermeerDmitriev.pdf
-          </a>
-        </p>
+        </div>
       </Modal>
-      <div className="alert alert-danger">
-        <strong>Warning: Sample Ratio Mismatch (SRM) detected</strong>. We
-        expected a <code>{formatTrafficSplit(expected, 1)}</code> split, but
-        observed a{" "}
-        <code>
-          {formatTrafficSplit(
-            observed,
-            getSRMNeededPrecisionP1(observed, expected)
+      {srm >= srmThreshold ? (
+        <div className="alert alert-info">
+          <b>
+            No Sample Ratio Mismatch (SRM) detected. p-value above{" "}
```

Comment:
Let's capitalize P-value when it starts a sentence. Awkward, but easier to read.